languages/de-DE/Docs/PaletteColours.multids

```diff
@@ -29,6 +29,8 @@ external-link-foreground-hover: Externer Link Vordergrund (hover)
 external-link-foreground-visited: Externer Link besucht Vordergrund
 external-link-foreground: Externer Link Vordergrund
 foreground: Vordergrund Global
+menubar-background: Menu bar Hintergrund
+menubar-foreground: Menu bar Vordergrund
```

Comment:
I think it's more like: "Top" Menü Hintergrund

Reply:
I purposely left out the word "top" in English because the intention is to support menubars at the bottom of the window, too. (That's much easier with the changes discussed in #4473)

Reply:
OK, I see. Then it's: Menü Hintergrund and Menü Vordergrund

Reply:
Ok @pmario @Jermolene

Reply:
I'd go with "Top Menü Hintergrund" and "Top Menü Vordergrund" then or "Oberes Menü Hintergrund" and "Oberes Menü Vordergrund" because Left/Bottom menus could be added

Reply:
@pmario, I've updated it to "Oberes Menü Hintergrund/Vordergrund" now